src/components/ViewOptions.tsx

```diff
@@ -73,7 +72,7 @@ export const ViewOptions: React.FC<{
         </PopoverContent>
       }
     >
-      {(onClick, ref, className) => (
+      {(onClick: any, ref: any, className: any) => (
```

Comment:
types

Reply:
This was erring when I got here, I can presume className is string but not sure what the others are exactly (or where they come from)